params/bootnodes.go

```diff
@@ -115,6 +115,8 @@ var KottiBootnodes = []string{
 	"enode://ae8658da8d255d1992c3ec6e62e11d6e1c5899aa1566504bc1ff96a0c9c8bd44838372be643342553817f5cc7d78f1c83a8093dee13d77b3b0a583c050c81940@18.232.185.151:30303",
 }
 
+var AstorBootnodes = []string{"", ""}
```

Comment:
Happy to run a bootnode if you want to start rolling out this network.

Reply:
On ETH node does the role of bootnodes, so if you have your public nodes running just copy their enodes here.

Reply:
Your error `Fatal: Error starting protocol stack: database already contains an incompatible genesis block (have 23af9941d03ec9be, new baa9bf2fa3454d87)` doesn't look like a bootnode issue to me; it looks like a chaindata db issue.